client/i18n/locales/espanol/intro.json

```diff
@@ -60,7 +60,7 @@
         "title": "Proyectos de diseño web responsivo",
         "intro": [
           "Es hora de poner tus habilidades recién aprendidas a funcionar. Al trabajar en estos proyectos, tendrás la oportunidad de aplicar todas las habilidades, principios y conceptos que has aprendido hasta ahora: HTML, CSS, diseño visual, accesibilidad y más.",
-          "Completa los cinco proyectos de programación web de abajo para obtener tu certificado de Diseño Web Responsivo."
+          "Completa los cinco proyectos de programación web de abajo para obtener tu certificación de diseño web responsivo."
```

Comment:
@RafaelDavisH I could be mistaken, but I thought you said at one point the only time a title would be capitalized is if it was a certification title.